src/crypto/tls/handshake_server_tls13.go

```diff
@@ -84,6 +125,7 @@ func (hs *serverHandshakeStateTLS13) handshake() error {
 func (hs *serverHandshakeStateTLS13) processClientHello() error {
 	c := hs.c
 
+	hs.handshakeTimings.ReadClientHello = hs.handshakeTimings.elapsedTime()
```

Comment:
This should go after (the HRR and) CH has been processed, i.e., at the end of this function.

Reply:
Eh, I'm not sure. There's a lot of stuff in this function that happens as a result of processing the CH. But I'll move it now. We can shuffle it around later.